### PR TITLE
Assembling website

### PR DESCRIPTION
**Pull request by dalshekerchi**
- Worked with bootstrap to set up website

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -39,8 +40,64 @@ body {
...
+    position: absolute;
+    top: -2px; left: -2px;
+    bottom: -2px;  right: -2px;
     background: linear-gradient(90deg, #4458dc 0%, #854fee 100%);
-    -webkit-background-clip: text;
-    -webkit-text-fill-color: transparent;
-    background-clip: text;
-  }
\ No newline at end of file
+    content: '';
```
Comment (pavpen) [inline portfolio/src/main/webapp/style.css]: Question: If I'm reading this correctly, adding an empty text node at the end of an element with class `primary-button` would have no visible effect. Is this `.primary-button::after` CSS rule really needed?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -87,17 +87,44 @@ h4 {
...
+.about-me {
+  padding-top: 20px;
+}
+
+.about-img {
+  width: 80%;
+}
+
+.about-me::after {
+    content: " ";
```
Comment (pavpen) [inline portfolio/src/main/webapp/style.css:119]: Q: I don't quite understand the purpose of using CSS to add a space after an element either. There may be simpler way to do what this CSS rule is trying to achieve.

**Review by pavpen — APPROVED** [review]
Added a few comments, some of which are questions to clarify the intended purpose of some of the code.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,18 +2,114 @@
 <html>
```
Comment (andy-keene) [inline portfolio/src/main/webapp/index.html:2]: This deserves a special shoutout for being one of the best designed sites I've seen as a PA in SPS!
 👏 🎈 🥇

**Review by andy-keene — APPROVED** [review]
These changes are good to merge.

**Pull request merged**